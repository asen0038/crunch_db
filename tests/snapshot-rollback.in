set a 2 4 1 9 1
snapshot
sort a
uniq a
get a
rollback 1
get a
bye
