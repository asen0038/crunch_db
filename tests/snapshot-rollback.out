>
ok

>
saved as snapshot 1

>
ok

>
ok

>
a [1 2 4 9]

>
ok

>
a [2 4 1 9 1]

>
bye
